www/_templates/site.html

```diff
@@ -51,7 +51,7 @@
                 <li><a href="https://blog.pyladies.com">Blog</a></li>
                 <li><a href="{{ get_url('CodeOfConduct/') }}">Code of Conduct</a></li>
                 <li><a href="{{ get_url('resources/') }}">Resources</a></li>
-                <li><a href="mailto:info@pyladies.com?subject=Hello" id="contact-link">Contact</a></li>
+                <li><a href="https://github.com/pyladies/info/issues/new" id="contact-link">Contact</a></li>
```

Comment:
```suggestion
                <li><a href="https://github.com/pyladies/info/" id="contact-link">Contact</a></li>
```

I think it would be great for them to land to the FAQ first before opening an issue. So I suggested link change.